ingress/ingress-nginx-dep.yaml

```diff
@@ -37,16 +47,11 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
-          resources:
```

Comment:
Why remove the resource requirement?

Reply:
we should keep that! I've lowered it now thou, looking into our history it never moves above 60 MB